Jenkinsfile.kei.beta

```diff
@@ -0,0 +1,40 @@
+#!groovy
+
+pipeline {
+  agent { 
+    label 'docker'
+  }
+
+  options {
+    ansiColor('xterm')
+  }
+
+  environment {
+    AWS_DEFAULT_REGION = 'ap-southeast-2'
+  }
+
+  stages {
+
+    stage('Deploy') {
+      steps {
+        withCredentials([
+        [
+          $class: 'AmazonWebServicesCredentialsBinding',
+          credentialsId: 'jenkins-section-website',
+        ],
+        usernamePassword(credentialsId: 'HARRY_DOCKER_HUB_USER', usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD'),
+        string(credentialsId: 'www.section.io-kei-user-token',
+          variable: 'KEI_USER_TOKEN')
+        ]) {
+          // echo "KEI_USER_TOKEN: ${env.KEI_USER_TOKEN}"
+          sh 'echo "$DOCKER_PASSWORD" | docker login --username=$DOCKER_USERNAME --password-stdin'
+          sh 'curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.19.6/bin/linux/amd64/kubectl && \
+              chmod +x ./kubectl && \
+              mv ./kubectl /usr/local/bin/kubectl'
+          sh 'make deployKEIBeta'
+          sh 'kubectl apply -k ../k8s/base'
```

Comment:
Just a note for later, we'll want to apply `../k8s/beta` once its created. Seems good to test with the base for now though.